.github/workflows/docker-images.yml

```diff
@@ -237,7 +240,7 @@ jobs:
           && echo "AMORO_VERSION=${AMORO_VERSION}" >> $GITHUB_OUTPUT
 
       - name: Build optimizer module with Maven
-        run: mvn clean package -pl 'amoro-optimizer/amoro-optimizer-spark' -am -e ${OPTIMIZER_SPARK} -DskipTests -B -ntp
+        run: mvn clean package -pl 'amoro-optimizer/amoro-optimizer-spark' -am -e ${OPTIMIZER_SPARK} -DskipTests -B -ntp -Djava.version=${{ matrix.jdk }}
```

Comment:
Do we need to set `maven.compiler.source` and `maven.compiler.target` in the command line?

executed on my local machine with `mvn clean package -Djava.version=11`, but it did not run with jdk11, maybe we need to set the properties `maven.compiler.{source/target}`?